### ISSUE TITLE
Test failure in latest

### ISSUE THREAD

**Issue by BridgeAR** [cheeriojs/cheerio#1284]
When running the tests there is a test failure. For that reason this module shows up on CITGM as failing (https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/1758/nodes=ppcle-ubuntu1404/testReport/junit/(root)/citgm/cheerio_v1_0_0_rc_2/).

```
   608 passing (2s)
   1 pending
   1 failing
   1) cheerio .load should render xml in html() when options.xml = true passed to html():
      Error: expected '<html><head></head><body><mixedcasetag uppercaseattribute></mixedcasetag></body></html>' to equal '<html><head></head><body><mixedcasetag uppercaseattribute=""></mixedcasetag></body></html>'
       at Assertion.assert (node_modules/expect.js/index.js:96:13)
       at Assertion.be.Assertion.equal (node_modules/expect.js/index.js:216:10)
       at Assertion.(anonymous function) [as be] (node_modules/expect.js/index.js:69:24)
       at Context.<anonymous> (test/cheerio.js:375:29)
```

It would be great if you could publish a new version soon without the test failure.

**Comment by jugglinmike** [comment]
This is an intentional bug fix for https://github.com/cheeriojs/cheerio/issues/702; we neglected to update Cheerio's tests because the fix come through a dependency. We should have a correction for this in a new release candidate shortly--see https://github.com/cheeriojs/cheerio/pull/1291. Sorry for the interruption!

**Comment by BridgeAR** [comment]
@jugglinmike @fb55 @davidchambers thanks a lot for fixing the test but please also release a new version. Cheerio has long been noise in CITGM and it is very hard to reason with the output if there's too much noise.

**Comment by BridgeAR** [comment]
@jugglinmike @fb55 we had to remove cheerio from CITGM (the tests won't be run anymore). Please let us know when the fixed test suite is released or better: open a PR in CITGM to activate it again, thanks.

**Comment by jugglinmike** [comment]
Published and tagged as [1.0.0-rc.3](https://github.com/cheeriojs/cheerio/tree/1.0.0-rc.3):

    $ npm show cheerio | tail -n 4
    dist-tags:
    latest: 1.0.0-rc.3
    
    published just now by jugglinmike <mike@mikepennisi.com>

Once again, my apologies for the interruption

**Comment by BridgeAR** [comment]
@jugglinmike thanks for following up on this! :heart:
